closure/goog/ui/datepicker.js

```diff
@@ -1423,7 +1464,13 @@ goog.ui.DatePicker.prototype.redrawCalendarGrid_ = function() {
   }
 
   var month = this.activeMonth_.getMonth();
-  var today = new goog.date.Date();
+  var today;
```

Comment:
Yea, I hate redundant code too. I'll abstract this out to a function and add it in my next commit.